frontend/src/components/PizzaList.js

```diff
@@ -1,22 +1,70 @@
 import Pizza from "./Pizza";
 import { Container, Row, Col } from "react-bootstrap";
 import "bootstrap/dist/css/bootstrap.min.css";
+import { useState, useEffect } from "react";
 
 const PizzaList = (props) => {
   const listOfPizzas = props.listOfPizzas;
   const onAdd = props.onAdd;
+  const categories = ["classic", "supreme", "signature"]; //try thiis
```

Comment:
Are these your pizzas or just types of Pizzas?

Reply:
These are types of pizzas. Each pizza is categorized into either of the three types.

Reply:
Thanks!